pkgs/development/libraries/science/math/magma/generic.nix

```diff
@@ -8,10 +8,11 @@
 { blas
 , cmake
 , cudaPackages
-, cudaSupport ? true
+, cudaSupport ? true # Note: this way ofborg won't evaluate magma -> won't fail
```

Comment:
why does this prevent ofborg from evaluating magma? shouldn't it evaluate successfully for both `cudaSupport = true` and `cudaSupport = false`?